### PR TITLE
Backport of the json stat fix

### PR DESCRIPTION
**Pull request by ydirson**

### CODE REVIEW

File: SPECS/xapi.spec [hunk 1]
```diff
@@ -79,7 +79,7 @@ BuildRequires: python2-devel
 %endif
 
 BuildRequires: python3-devel
-BuildRequires: xs-opam-repo >= 6.54.0-2
+BuildRequires: xs-opam-repo > 6.74.0-1.1
```
Comment (benjamreis) [inline SPECS/xapi.spec:82]: It's a strictly superior operator, is it wanted?
Especially since you mention this exact version in the changelog comment.

Comment (ydirson) [reply SPECS/xapi.spec:82]: This is intended for the testing phase, while `6.74.0-1.2` is not issued yet.  But `yum-builddeps` happily ignores it, so we'd better prepare for it already.

Comment (stormi) [reply SPECS/xapi.spec:82]: The intention is good, but this line will conflict as soon as XenServer updates it and I know I'll likely be the one handling the conflict :)

I'll opt for a neutral approach in my review regarding this line.

**Review by stormi — COMMENTED** [review]
LGTM, but I'll only approve when the release number and changelog are final

File: SPECS/xapi.spec (hunk 1 above)
Comment (ydirson) [reply SPECS/xapi.spec]: done

**Review by benjamreis — APPROVED** [review]

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
Koji build (v8.3-incoming): http://koji.xcp-ng.org/taskinfo?taskID=47293

**Pull request merged**